R/tuneMBO.R

```diff
@@ -14,6 +14,11 @@ tuneMBO = function(learner, task, resampling, measures, par.set, control,
   force(opt.path)
   force(show.info)
 
+  if (!is.null(mbo.control$n.objectives)) {
```

Comment:
put that below and just `if (multicrit) {...}` ?

Reply:
I will do if (multicrit) but I think that this check should take place before the actual tuning.
Also, I think now that !is.null(...) is not needed at all. mlrMBO makes sure that n.objectives is an integerish number.